doc/ReleaseNotes.md

```diff
@@ -8,6 +8,7 @@ The release versions are NuGet releases.
 * check if BaseUri is absolute (see [PR #738](https://github.com/vvvv/SVG/pull/738))
 
 ### Fixes
+* fixed the parsing of significant whitespace (see [#744](https://github.com/vvvv/SVG/issues/744) and [PR #744])(https://github.com/vvvv/SVG/pull/745))
```

Comment:
PR #745

Reply:
And remove close bracket.

`[PR #745](`

Reply:
Fixed it - apologies for my sloppiness